tests/libvmi/vmi.go

```diff
@@ -252,10 +252,14 @@ func WithPasstInterfaceWithPort() Option {
 }
 
 func WithNodeAffinityFor(nodeName string) Option {
+	return WithNodeAffinityForLabel("kubernetes.io/hostname", nodeName)
+}
+
+func WithNodeAffinityForLabel(nodeLabelKey, nodeLabelValue string) Option {
```

Comment:
Please relocate to a dedicated file (see suggested rules in the readme), vmi.go should not be populated unless there is a very good reason.

This may fit into `selector` or `nodeselector`, or anything you think fits better to describe this group of options.